zubhub_frontend/zubhub/src/assets/css/index.css

```diff
@@ -60,15 +74,15 @@ html,
 }
 
 .dropdown-menu {
-  background-color: #ffcc00;
+  background-color: 'var(--primary-color2)';
 }
 
 .gallery-sub-nav {
   font-size: calc(10px + 3vh);
   font-weight: 900;
   display: flex;
   justify-content: space-between;
-  background-color: #ffcc00;
+  background-color: 'var(--primary-color2)';
```

Comment:
We might also need to set foreground colors wherever we are setting background colors? 
Can you check how it looks if we set dark vs light colors